cypress/pages/slices/editWidgetModals/geoPointModal.js

```diff
@@ -0,0 +1,3 @@
+import { BaseEditWidgetModal } from "./baseEditWidgetModal";
+
+export class GeoPointModal extends BaseEditWidgetModal {}
```

Comment:
how come this one is empty?

Reply:
Doesn't add any additional functionality from the parent class it inherits